### PR TITLE
added swagger-PS-feature

### PR DESCRIPTION
**Pull request by serhii-shyian**

### CODE REVIEW

File: src/main/java/com/example/bookstore/controller/BookController.java [hunk 1]
```diff
@@ -20,43 +23,56 @@
...
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
-    @ResponseStatus(HttpStatus.OK)
     @GetMapping("/search")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Get all books by parameters",
+            description = "Get a list of all books according to the parameters")
     public List<BookDto> searchBooks(BookSearchParametersDto searchParameters) {
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/bookstore/controller/BookController.java]: Add `@ParameterObject @PageableDefault Pageable pageable`

File: src/main/java/com/example/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -20,43 +23,56 @@
...
     private final BookService bookService;
 
-    @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Get all books",
+            description = "Get a list of all available books")
+    public List<BookDto> getAll(Pageable pageable) {
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/bookstore/controller/BookController.java]:
```suggestion
    public List<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {
```

File: src/main/java/com/example/bookstore/model/Book.java [hunk 3]
```diff
@@ -37,5 +38,6 @@ public class Book {
     private String description;
     private String coverImage;
     @Column(nullable = false)
-    private boolean isDeleted = false;
+    @Value("false")
+    private boolean isDeleted;
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/bookstore/model/Book.java:40]: Remove `@NoArgsConstructor`

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see pr

**Review by okuzan — APPROVED** [review]

**Pull request merged**